src/main/java/ua/danit/final_project/entities/Employee.java

```diff
@@ -0,0 +1,47 @@
+package ua.danit.final_project.entities;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.OneToOne;
+import javax.persistence.FetchType;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'javax.persistence.FetchType'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=39&issues=AWU3cB7eW2AOP0CkE0NI&open=AWU3cB7eW2AOP0CkE0NI)

[ISSUE_KEY]: AWU3cB7eW2AOP0CkE0NI